go/service/main.go

```diff
@@ -227,6 +233,18 @@ func (d *Service) checkTrackingEveryHour() {
 	}()
 }
 
+func (d *Service) gregordConnect() error {
+	h := newGregorHandler(d.G())
+	d.G().Log.Debug("gregor URI: %s", d.G().Env.GetGregorURI())
+	rootCert, err := ioutil.ReadFile(os.Getenv("GREGOR_ROOT_CERT"))
```

Comment:
Let's follow the example of what we do in `client.go` for connecting to the main API endpoint, to keep tings consistent.